test/CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ if(NOT (MIOPEN_TEST_GFX900 OR MIOPEN_TEST_GFX906 OR MIOPEN_TEST_GFX908 OR MIOPEN
     endif()
 endif()
 
-set_var_to_condition(MIOPEN_LIBMLIR_SUPPORTS_GFX103X_DEFAULT ${MIOPEN_USE_MLIR} AND (${rocMLIR_VERSION_FLAT} GREATER 100000000))
+set_var_to_condition(MIOPEN_LIBMLIR_SUPPORTS_GFX103X_DEFAULT ${MIOPEN_USE_MLIR} AND (${rocMLIR_VERSION} VERSION_GREATER_EQUAL 1))
```

Comment:
@apwojcik MLIR is an optional component of MIOpen, thus `rocMLIR_VERSION` is not guaranteed to be defined here unfortunately.